problems/variables.js

```diff
@@ -16,14 +22,28 @@
 //  * Calculate how many snacks you would eat total, from your current age until the maximum age.
 //  * Log the result to the screen like this: "You will need `snackNumber` to last you until the age of `maxAge`".
 
-// ## Problem Three
+let curAge = "30"
+const maxAge = "99"
+let snackCount = "3"
+let snackNumber = ((maxAge - curAge)*(snackCount * 365))
+
+console.log("You will need "+snackNumber+ " snacks to last you, until the age of "+maxAge)
 
+
+// ## Problem Three
 // Calculate properties of a circle, using the definitions: http://math2.org/math/geometry/circles.htm 
 //  * Store a radius into a variable.
 //  * Calculate the circumference based on the radius, and log "The circumference is `circumferenceResult`".
 //  * Calculate the area based on the radius, and log "The area is `areaOfCircle`".
 //  * Hint: https://www.w3schools.com/jsref/jsref_pi.asp
 
+
+
+let radius = "2"
+let circumferenceOfCircle = Math.PI * radius 
+let areaOfCircle = Math.PI * (radius * 2)
```

Comment:
I like how you're saving equations in variables, but look at the link provided for properties of a circle. Your equations are not correct.